nodejs/sqlcommenter-nodejs/packages/sqlcommenter-sequelize/provider/opentelemtryprovider.js

```diff
@@ -0,0 +1,40 @@
+// Copyright 2019 Google LLC
```

Comment:
Please use 2020 for the copyright year since that's when this code is being added.
s/2019/2020/g

Also the file name spells "opentelemtry" instead of "opentelemetry", please update that, and we can remove the provider.js suffix since it is verbose.